### PR TITLE
QField should reflect the avatar of the logged in cloud user

### PR DESCRIPTION
**Pull request by nirvn**
Yes, the most crucial feature has landed in QField, avatar image!
image

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1d06d5b6de902380d9da4240a6db677eef9fb6b1:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-8e0c1a86b6191646f17ee5f00baecb19a06886e5-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-8e0c1a86b6191646f17ee5f00baecb19a06886e5-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-8e0c1a86b6191646f17ee5f00baecb19a06886e5-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-8e0c1a86b6191646f17ee5f00baecb19a06886e5-x86.apk)

File: src/core/qfieldcloudconnection.cpp [hunk 1]
```diff
@@ -246,7 +251,11 @@ void QFieldCloudConnection::login()
 
     mUsername = resp.value( QStringLiteral( "username" ) ).toString();
     emit usernameChanged();
-    mUserInformation  = CloudUserInformation( mUsername, resp.value( QStringLiteral( "email" ) ).toString() );
+    mAvatarUrl = resp.value( QStringLiteral( "avatar_url" ) ).toString();
+    emit avatarUrlChanged();
+    qDebug() << resp.value( QStringLiteral( "avatar_url" ) ).toString();
```
Comment (suricactus) [inline src/core/qfieldcloudconnection.cpp:256]: debug

File: src/core/qfieldcloudconnection.cpp [hunk 2]
```diff
@@ -246,7 +251,11 @@ void QFieldCloudConnection::login()
 
     mUsername = resp.value( QStringLiteral( "username" ) ).toString();
     emit usernameChanged();
-    mUserInformation  = CloudUserInformation( mUsername, resp.value( QStringLiteral( "email" ) ).toString() );
```
Comment (suricactus) [inline src/core/qfieldcloudconnection.cpp:249]: This was added by @signedav , but if you think it is not critical, we can remove it. He still has debts to pay for the changelog :smiling_imp:

**Review by suricactus — APPROVED** [review]
A small commit for QField, a giant leap for QFieldCloud.

**Comment by suricactus** [comment]
btw, probably adding some 1-2 px border in super green of the round image avatar to indicate it is clickable would be nice?

File: src/core/qfieldcloudconnection.cpp (hunk 2 above)
Comment (signedav) [reply src/core/qfieldcloudconnection.cpp:249]: You are a cruel human being 😉

All good.

**Comment by nirvn** [comment]
@suricactus , comments addressed, border added.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 710afc2f41f419ca9198a3b7df3062d8f1ac423f:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-99902cd0577fa843e6a0e97850e1f611cee670f6-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-99902cd0577fa843e6a0e97850e1f611cee670f6-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-99902cd0577fa843e6a0e97850e1f611cee670f6-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1840-99902cd0577fa843e6a0e97850e1f611cee670f6-x86.apk)

**Review by suricactus — APPROVED** [review]

**Pull request merged**